docs/source/what-is-mlflow.rst

```diff
@@ -0,0 +1,51 @@
+.. _what-is-mlflow:
+
+What is MLflow?
+===============
+
+MLflow is a versatile, expandable, open-source platform for managing machine learning workflows and artifacts across the machine learning lifecycle. It has predefined flavors for many popular ML libraries, but can be used with any library, algorithm, or deployment tool. It is designed to be extensible, so you can write plugins to support new workflows, libraries, and tools.
+
+.. figure:: _static/images/what-is-mlflow/mlflow-overview.png
+    :width: 100%
+
+MLflow is composed of five components:
```

Comment:
The list here is essentially repeated under "What is MLflow used for?" Is it possible to combine the two sections?

Maybe a table with columns? | MLflow component | What it does |